src/App.tsx

```diff
@@ -1,10 +1,26 @@
-import React from 'react';
+import React, { useEffect, useState } from "react";
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [pressedKey, setPressedKey] = useState("");
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  const keyBoardHandler = (event: KeyboardEvent) => {
```

Comment:
Keep this rule 
![image](https://user-images.githubusercontent.com/122989585/219306862-6a475c6a-b4a4-4b34-9fdf-7440658ee4c9.png)